pyaerocom/aeroval/modelmaps_helpers.py

```diff
@@ -85,36 +114,28 @@ def calc_contour_json(data, cmap, cmap_bins):
     lats = data.latitude.points
     lons = data.longitude.points
 
-    geojson = {}
+    geojsons = {}
     tst = _jsdate_list(data)
+    colors_hex = [to_hex(val) for val in cm.colors]
     for i, date in enumerate(tst):
         datamon = nparr[i]
-        contour = ax.contourf(
-            lons,
-            lats,
-            datamon,
-            transform=proj,
-            colors=cm.colors,
-            levels=cmap_bins,
-            extend="max",
-        )
-
-        result = contourf_to_geojson(contourf=contour)
-
-        geojson[str(date)] = eval(result)
+        levels = cmap_bins
+        # Top contour should include everything above seconds to last value
+        levels[-1] = np.inf
 
-    ax.figure.colorbar(contour, ax=ax)
-    colors_hex = [to_hex(val) for val in cm.colors]
+        geojson = _filled_contours_to_geojson_features(
+            lons, lats, datamon, levels=cmap_bins, properties=[{"fill": val} for val in colors_hex]
```

Comment:
@AugustinMortier What properties do you base the colours of the contours on? I've set the colour on each contour here in the hopes that the rendering can use this directly

Reply:
I do use the fill property: https://gitlab.met.no/aeroval/web/-/blob/master/src/modules/maps.ts?ref_type=heads#L765